src/openforms/forms/api/viewsets.py

```diff
@@ -183,7 +183,7 @@ def get_serializer_class(self):
 
     @extend_schema(
         summary=_("Retrieve form definition JSON schema"),
-        tags=["forms"],
+        tags=["forms", "form-definitions"],
```

Comment:
doesn't this cause the same endpoint to be listed in multiple places? we don't want that

Reply:
This is already the case for a bunch of other endpoints, actually all the `FormDefinitionViewSet` endpoints